### PR TITLE
Update Configuration.MD

### PR DESCRIPTION
**Pull request by eisraeli**
Rephrase the base_images description in order to make it less confusing.

### CODE REVIEW

File: CONFIGURATION.md [hunk 1]
```diff
@@ -130,9 +130,9 @@ override specific components for specific tests at non-standard levels when many
 streams and one tag are used to assemble a release.
 
 # `base_images`
-`base_images` are parents for images built from the repository, but are not built
-from the repository. The field is a mapping from pipeline image name to remote
-`ImageStream` specification. A common base image might be an operating system:
+`base_images` are parent images.
```
Comment (petr-muller) [inline CONFIGURATION.md]: I feel like we are losing information with this change (parent images for what?). I agree that the original sentence is not ideal. 

Maybe "... are external images that serve as parents for images built from the repository"?

Comment (eisraeli) [reply CONFIGURATION.md]: @petr-muller 
Yeah, I agree. That sounds better.

**Comment by eisraeli** [comment]
@petr-muller It should be more clear now without losing any information. Thanks.

**Pull request merged**